Makefile

```diff
@@ -32,14 +31,15 @@ inplace:
 	$(PYTHON) setup.py build_ext -i
 
 test-code:
-	$(NOSETESTS) -s nilearn $(NOSETESTS_OPTIONS)
+	python -m $(TEST_RUNNER) --pyargs nilearn --cov=nilearn -s -vv --durations=0
+
 test-doc:
-	$(NOSETESTS) -s --with-doctest --doctest-tests --doctest-extension=rst \
-	--doctest-extension=inc --doctest-fixtures=_fixture `find doc/ -name '*.rst'`
+	$(TEST_RUNNER) -s --doctest-glob='*.rst' --doctest-modules
```

Comment:
```suggestion
	$(TEST_RUNNER) $(find -name "*.rst")
```

Reply:
Whyy are you giving me code samples for everything in this PR? :-)